dotcom-rendering/src/web/components/CalloutEmbed/Form.test.tsx

```diff
@@ -230,31 +219,30 @@ describe('Callout from', () => {
 			[checkboxField.id]: ['checkbox 1', 'checkbox 3'],
 		});
 	});
-	test('should upload the file', () => {
-		const file = new File(['hello'], 'hello.png', { type: 'image/png' });
-		const mockSubmit = jest.fn();
-		const { queryByText } = render(
-			<Form formFields={[fileField]} onSubmit={mockSubmit} />,
-		);
-
-		const input = screen.getByTestId<HTMLInputElement>(
-			`form-field-${fileField.id}`,
-		);
-		user.upload(input, file);
-
-		const inputFiles = input.files ? input.files : [];
-
-		expect(inputFiles[0]).toStrictEqual(file);
-		expect(inputFiles).toHaveLength(1);
-
-		const submitButton = queryByText(
-			'Share with the Guardian',
-		) as HTMLButtonElement;
-		fireEvent.click(submitButton);
-
-		expect(mockSubmit.mock.calls.length).toBe(1);
-		// TODO: test mockSubmit internal
-	});
+	// test('should upload the file', () => {
+	// 	const file = new File(['hello'], 'hello.png', { type: 'image/png' });
```

Comment:
Something odd happens here where the file is never created instead file = `{}`

Reply:
Should you use `test.skip`, here?